### DISCUSSION TITLE
Use notify to get 2 arrays in python app

### DISCUSSION THREAD

**Discussion by ImmanuelT8** [hbldh/bleak#1298] [Q&A] [answered]
* bleak version: last
* Python version: 3.10
* Operating System: windows

### Description

Hi, 

i want to show 2 arrays in my python App. 
The ESP code works and the code in the App too, 
but i only can print 1 parameter with:

```python
async def notification_handler(sender, data):
    value = numpy.frombuffer(data, dtype=numpy.uint32)
    print("red:")
    print(value)
    
async def notification_handler_ir(sender, data_ir):
    ir_value = numpy.frombuffer(data, dtype=numpy.uint32)
    print("IR Wert:")
    print(ir_value)
```
   
How i can use the function notification_handler for 2 arrays?

Thx for help.

**Comment by dlech** [comment]
If you use two different characteristics, each characteristic can have it's own notification handler.

**Reply by ImmanuelT8** [reply]
Thanks for your reply! Can you show me an example code? Regards

**Reply by dlech** [reply, marked as answer]
```python
client.start_notify(UUID1, handle_uuid1_notify)
client.start_notify(UUID2, handle_uuid2_notify)
```

**Reply by ImmanuelT8** [reply]
Work, thank you very much!
